src/ecu/context/ECUContext.tsx

```
@@ -167,108 +218,160 @@ export const ECUProvider: FC<ECUProviderProps> = ({ children }) => {
+              const currentState = ecuStore.getState();
+              if (
+                currentState.status === ECUConnectionStatus.CONNECTED &&
+                (currentState.detectedEcuAddresses?.length ?? 0) > 0
+              ) {
+                clearTimeout(timeout);
+                unsubscribe();
+                resolve();
+              }
+            });
+          });
+        } catch (timeoutError) {
```

ESLint

[error] 'timeoutError' is defined but never used.

(@typescript-eslint/no-unused-vars)